internal/store/pod.go

```diff
@@ -780,40 +884,23 @@ var (
 					req := c.Resources.Requests
 
 					for resourceName, val := range req {
-						switch resourceName {
-						case v1.ResourceCPU:
+						if isHugePageResourceName(resourceName) {
```

Comment:
Can you explain the below change a bit more, you wanted to just do the breaking changes around the units, why all these changes?

Reply:
This code doesn't change anything, this just extracted the `default` path from the `switch` statement, as all the other cases now have explicit metrics again.